nservicebus/licensing/index_license-management_core_[7,8).partial.md

```diff
@@ -20,19 +20,15 @@ A license located at `{AppDomain.CurrentDomain.BaseDirectory}/license.xml` will
 
 ### User-specific license location
 
-To install a license for all endpoints and Particular Service Platform applications run by a specific user, install the license file to `{SpecialFolder.LocalApplicationData}\ParticularSoftware\license.xml`.
-
-This location can be expressed using environment variables on Windows, or a bash expression on Linux/macOS:
```

Comment:
There was some previous conversation, I was unable to find the reference that you mentioned to PARTICULARSOFTWARE_LICENSE ENV variable. I that the document being updated makes that explicit anywhere. That could be added. But the text being changed wasn't referring to that ENV variable.

Reply:
Ah now I understand this part better. The environment variables or bash expression was referring to the bullet points below. So that fact that we have not yet documented `PARTICULARSOFTWARE_LICENSE` is basically a gap in the current documentation and not related to this change. Thanks for the patience with me :dancers:

Reply:
Well, given this new source of info, @danielmarbach @awright18 should I add the `PARTICULARSOFTWARE_LICENSE` to the docs?
And if so, which version is this available in?

Reply:
I'll create another PR.